src/lib/bootloader/grub2base.rb

```diff
@@ -217,11 +212,15 @@ def merge_grub_default(other)
     end
 
     def merge_attributes(default, other)
-      # string attributes
-      [:serial_console, :terminal, :timeout, :hidden_timeout, :distributor,
-       :gfxmode, :theme, :default].each do |attr|
-        val = other.public_send(attr)
-        default.public_send((attr.to_s + "=").to_sym, val) if val
+      begin
+        # string attributes
+        [:serial_console, :terminal, :timeout, :hidden_timeout, :distributor,
+         :gfxmode, :theme, :default].each do |attr|
+          val = other.public_send(attr)
+          default.public_send((attr.to_s + "=").to_sym, val) if val
+        end
+      rescue RuntimeError
```

Comment:
Um, I'm not sure if catching all run time exceptions is a good approach here. Maybe a comment would help here...

Reply:
yep, but cfa_grub2 raise generic RuntimeError, I hope it is just temporary solution. Also cfa_grub2 can be improved to raise specific exception, but currently for SLE15 GA I do not want to touch it.

Reply:
OK, then just add comment with that info.